doc/man/cockpit.conf.xml

```diff
@@ -74,6 +74,42 @@ Origins = https://somedomain1.com https://somedomain2.com:9090
     </variablelist>
   </refsect1>
 
+  <refsect1 id="oauth">
+	  <title>oauth</title>
```

Comment:
Hmmm, the actual mechanisms in the headers are typically 'Basic' etc... I think we should change things the other way around. I think we are case insensitive here, right? But we should document stuff consistently. The CamelCase came from systemd config style, by the way.